Feature/bound 36 Rules Optional Trigger

EthanThomas0523 opened the pull request:
Moved the rules function into the game setup class and created an optional trigger to print the rules

EthanThomas0523 committed:
Added an Optional Trigger to read the rules

SCustodero commented on source/CardGames.py:
```diff
@@ -433,12 +446,7 @@ def marshall_first_turn(self):
       self.display_board_general()
 
 game = GameSetup()
+game.ReadRules()
```
Cool stuff. Could you call ReadRules() in the first line of the start_game() function instead of at the bottom just to keep as much as we can contained in start_game()?

EthanThomas0523 replied:
Thanks, Sean! I will be sure to move the call for ReadRules() to the first line of the start_game() function.

EthanThomas0523 committed:
Moved the call for ReadRules

Moved to the beginning of the start_game function

youraveragedeveloper7 commented on source/CardGames.py:
```diff
@@ -190,8 +190,22 @@ def __init__(self):
+
+    def ReadRules(self):
+        choice = ""
+        while choice != "1":
```
It might be better to use an OR statement such as choice != "1" or choice != "0" to get rid of the if statement within the while loop.

EthanThomas0523 replied:
Thanks, Ian! I am not sure why I used an if statement instead of an or statement in the first place. I will be sure to change it.

EthanThomas0523 replied:
Thanks, Ian! I am not sure why I used an if statement instead of an and statement in the first place. Also, I believe it should be and instead of or since it would go in an infinite loop if it was or.  I will be sure to change it.

EthanThomas0523 committed:
Read Rules Optional Trigger Change

Changed the if statement in the while loop of the ReadRules function to an and statement in the condition for the while loop

EthanThomas0523 committed:
Merge branch 'development/The-Boundless' into feature/bound-36-Rule-Function
